packages/@angular/cli/commands/completion.ts

```diff
@@ -117,14 +117,15 @@ const CompletionCommand = Command.extend({
         });
       }
 
-      let opts = '';
+      let opts: string[] = [];
       if (command.blueprints && command.blueprints[0]) {
-        opts += extractBlueprints(command.blueprints);
+        opts = opts.concat(extractBlueprints(command.blueprints));
       }
 
       if (command.availableOptions && command.availableOptions[0]) {
```

Comment:
Let agree to disagree.

The 'installation' suggestion helps temporarily.

I am looking for the long-term benefit, by sourcing the output of `ng completion --shell`.
This is a lot more stable, because it is at least compatible with the installed ng version.

I can see people opening github issues "Completion broken, new option not supported" etc.